### PR TITLE
Read me

### PR DESCRIPTION
**Pull request by mesielepush**
Pull request to merge feature to Development. All fixes to the last pull request were implemented. Branches now follow the git-flow, and Read me file it's done.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -85,14 +85,7 @@ <h4 class="modal-title white-text w-100 font-weight-bold py-2">New Book</h4>
...
                 <div class="field">
                   <label class="label">Read Status</label>
-                  <div class="control">
-                    <div class="select">
-                      <select name="status" id="status">
-                        <option value="true">Read</option>
-                        <option value="false">Not read</option>
-                      </select>
-                    </div>
-                  </div>
```
Comment (maelfosso) [inline index.html]: It seems like you remove this. Why?
I think it caused the error mentioned above.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @mesielepush 

Thank you for this nice introduction to this project.
Your UI/UX is too simple. That largely simplify the navigation
However there are some issues to fix

1. From the form to add a book, there's nothing for indicating the **Read** status

2. I got this error when wanting to save a new book
Screenshot_2020-06-26_19-10-28

Kindly fix those 2 issues to help us continuing reviewing your project.
Don't forget to submit a new code review request once done

Thanks

Mael FOSSO

**Comment by mesielepush** [comment]
Sorry for that, I removed a div to fix some bug and then forgot to put it back. It works now.

**Review by mcrd25 — APPROVED** [review]
# Changes Approved :heavy_check_mark: 

Hi @mesielepush :wave: 


Good job with changes! And since technically you have met ll the requirements I see np need for another 
review. 
However, I would encourage you to check out some optional changes:

##### OPTIONAL CHANGES
- IT does not matter what I put when I add a new book (read/not read) when it is added it always shows status as "Not read"
- I can currently add an empty book. This is not ideal
- Consider validation for a number of books (as it stands I can enter alphanumeric as opposed to only numeric into that field)


***If you have any questions, now or in the future, do not hesitate to contact me on social media, email or on slack and I will be glad to chat with you.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

**Pull request merged**